src/main/java/view/OutputView.java

```diff
@@ -0,0 +1,33 @@
+package view;
+import model.Car;
+import java.util.List;
+
+public class OutputView {
+    private static void printName(String carName) {
+        System.out.print(carName + " : ");
+    }
+
+    public static void printRoundResult(List<Car> cars) {
+        for (Car car : cars) {
+            printName(car.getCarName());
+            printPosition(car.getPosition());
+            System.out.println();
+        }
+        System.out.println();
+    }
+
+    public static void printRaceStartMessage() {
+        System.out.println("\n실행 결과");
+    }
+
+    private static void printPosition(int position) {
+        for (int i = 0; i < position; i++) {
+            System.out.print("-");
+        }
+    }
```

Comment:
```suggestion
    private static void printPosition(int position) {
        System.out.print("-".repeat(position));
    }
```

참고사항으로 더 간결하게 하자면 이런 방법도 있습니다 ㅋㅋ

Reply:
`printPosition` 메서드의 for 루프를 `"-".repeat(position)`을 이용해 변경했습니다. for문보다 훨씬 간결해져 좋은 것 같습니다!